concepts/zero-values/about.md

```diff
@@ -1,30 +1,57 @@
-# Zero Values
+# About
 
-Unlike some other languages, Go has no empty/null/undefined state for uninitialized variables. Every type has a [zero value][zero_values] that uninitialized variables will hold upon declaration.
+Go does not have a concept of empty, null, or undefined for variable values. Variables declared without an explicit initial value default to the zero value for their respective type.
 
-These default values are called the zero values for their respective types:
+The zero value for primative types such as booleans, numeric types, and strings are `false`, `0`, and `""`, respectively.
+
+The identifier `nil`, meaning zero, is the zero value for more complex types such as pointers, functions, interfaces, slices, channels, and maps.
+
+The following table details the zero value for Go's types.
 
 | Type      | Zero Value |
 | --------- | ---------- |
-| bool      | false      |
-| numeric   | 0          |
-| string    | ""         |
-| pointer   | nil        |
-| func      | nil        |
-| interface | nil        |
-| slice     | nil        |
-| channel   | nil        |
-| map       | nil        |
-
-The predeclared identifier `nil`, meaning zero, is the zero value for the more complex types in Go. Comparing a type whose zero value is not `nil` to `nil`, like string, is an error:
+| boolean   | `false`    |
+| numeric   | `0`        |
+| string    | `""`       |
+| pointer   | `nil`      |
+| function  | `nil`      |
+| interface | `nil`      |
+| slice     | `nil`      |
+| channel   | `nil`      |
+| map       | `nil`      |
+
+## Zero Value Construction
+
+The `var` keyword can be used to construct any type to its zero value:
+
+```go
+func main() {
+  var myBool bool
+  fmt.Println("Zero value boolean:", myBool)
+}
+```
```

Comment:
You can remove the main function from those examples if you want. But in any case this is missing a line below that shows the actual result, it is usually written like this on the Go track:  
`// Output: Zero value ....`

Sorry for all the nit picks, in general you did great work and I really appreciate your help!

Reply:
Never a nit pick! Thank you for the feedback.